### PR TITLE
Responsive Mobile Friendly Tables and Layouts

### PR DESCRIPTION
**Pull request by jimbrig**

### CODE REVIEW

**Comment by jimbrig** [comment]
Before:

image

After:

image

**Comment by jimbrig** [comment]
Orders Module:

image

**Comment by merlinoa** [comment]
Looks good for mobile.  I would set it to scrollX = TRUE when not on mobile, and only set the mobile layout when it is actually on mobile.  How does that sound?

Simple check for if browser opened from a mobile device: https://dev.to/timhuang/a-simple-way-to-detect-if-browser-is-on-a-mobile-device-with-javascript-44j3

File: shiny_app/www/js/is_mobile.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+$(document).on('shiny:sessioninitialized', function (e) {
+  var mobile = window.matchMedia("only screen and (max-width: 768px)").matches;
+  Shiny.onInputChange('is_mobile_device', mobile);
```
Comment (merlinoa) [inline shiny_app/www/js/is_mobile.js:3]: FYI - Shiny.setInputValue() is the newer version of Shiny.onInputChange.   You can use either one though.  It does not matter.

**Comment by merlinoa** [comment]
Cool.  I like it.  Please go ahead and merge after you have done some QA testing on an actual mobile device.  You can deploy to the dev cloud run to do the testing on your phone.

File: shiny_app/www/js/is_mobile.js (hunk 1 above)
Comment (jimbrig) [reply shiny_app/www/js/is_mobile.js:3]: Good to know, pulled that straight from https://stackoverflow.com/a/55506058 but your referenced article above is better and I will change javascript to `Shiny.setInputValue()` to be consistent.

**Review by jimbrig — COMMENTED** [review]
Done

**Pull request merged**